### PR TITLE
everywhere: fix typos

### PR DESCRIPTION
**Pull request by nazar01**
Fix a lot of typos

Signed-off-by: Nazar Kazakov <nazar.kazakov.work@gmail.com>

### CODE REVIEW

**Comment by nazar01** [comment]
Gotta admit it's quite funny :D

**Comment by nazar01** [comment]
Fixing some CI warnings...

**Comment by nazar01** [comment]
Ok, looks like the rest of CI warnings aren't meaningful

File: samples/boards/nrf/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c [hunk 1]
```diff
@@ -276,7 +276,7 @@ static int gen_onoff_status(struct bt_mesh_model *model,
 	return 0;
 }
 
-/* Generic Level (lIGHTNESS) Server message handlers */
+/* Generic Level (lightness) Server message handlers */
```
Comment (stephanosio) [inline samples/boards/nrf/mesh/onoff_level_lighting_vnd_app/src/mesh/device_composition.c:279]: to be consistent with `/* Generic Level (TEMPERATURE) Server message handlers */` below:

```suggestion
/* Generic Level (LIGHTNESS) Server message handlers */
```

**Review by stephanosio — CHANGES_REQUESTED** [review]
Looks ok overall. Just need to fix some.

**Review by ppryga-nordic — DISMISSED** [review]
Reviewed changes for BLE direction finding.
BTW. Thank for the corrections!

**Review by Laczen — DISMISSED** [review]
Changes to nvs are OK, thanks.

**Review by de-nordic — DISMISSED** [review]
VFS and mcumgr look ok.

**Review by ruuddw — DISMISSED** [review]
All ARC fixes look fine, thanks! Guess who will be top contributor for this release...

**Review by avisconti — DISMISSED** [review]
It's a tough job, but someone had to do it!

**Comment by Thalley** [comment]
Not sure why GH decided that I removed review request from all those people when I just removed it from myself :s

File: cmake/linker/ld/ld_script.cmake [hunk 2]
```diff
@@ -224,7 +224,7 @@ function(section_to_string)
     set(TEMP "${TEMP}\n  *(${name})")
     set(TEMP "${TEMP}\n  *(\"${name}.*\")")
   endif()
-
+#?
```
[the remark this answers is not in the document]
Comment (nazar01) [reply cmake/linker/ld/ld_script.cmake]: Fixed, thanks!

**Comment by nazar01** [comment]
Hmm, GH "dismissed stale reviews" for me, I didn't do it

**Comment by Thalley** [comment]
> Hmm, GH "dismissed stale reviews" for me, I didn't do it

That's due to a recent change in our GH settings. If you rebase your PR it will typically dismiss all current approvals.

**Comment by nazar01** [comment]
Rebased on main

**Review by ruuddw — APPROVED** [review]

**Review by nandojve — APPROVED** [review]

**Review by stephanosio — APPROVED** [review]

**Comment by gmarull** [comment]
@nazar01 did you use any tool to automate those changes? Would probably be good to add it to CI.

**Review by cfriedt — APPROVED** [review]

**Comment by nazar01** [comment]
> @nazar01 did you use any tool to automate those changes? Would probably be good to add it to CI.

It was mostly manual, I went through all spellcheck warnings in my IDE (CLion)

**Review by microbuilder — APPROVED** [review]

**Comment by nazar01** [comment]
Merge?

**Comment by nazar01** [comment]
To whoever wants to merge it: feel free to delete my merge commit and rebase this PR on main instead.

**Comment by gmarull** [comment]
> To whoever wants to merge it: feel free to delete my merge commit and rebase this PR on main instead.

You should rebase and force-push. It looks like CI has some failures (maybe the changes triggered some unknown issues), if they are not related to your changes, please report them.

**Pull request merged**